opensrp-app/src/main/java/org/smartregister/ssl/OpensrpSSLHelper.java

```diff
@@ -61,4 +70,27 @@ public void verify(String host, String[] cns, String[] subjectAlts) throws
         }
     }
 
+    public javax.net.ssl.SSLSocketFactory getSSLSocketFactory() {
```

Comment:
Is it possible for us to enforce what versions of TLS the socket should expect? Since we're [recommended](https://latacora.singles/2018/04/03/cryptographic-right-answers.html) to use only 1.2 or higher.

Reply:
i.e the client should not accept connections if TLS version is 1.1 or less.

Reply:
its possible to use `TLSv1.2`

Reply:
Cool. Is it also possible to use TLS 1.0? If it is, then it would be a nice idea to reject such a connection.

Reply:
Cool, got you.